clouds/aws/aws_utils.py

```diff
@@ -139,3 +139,35 @@ def delete_bucket(bucket_name: str, boto_client) -> None:
         f"Bucket {bucket_name} not deleted",
         json.dumps(response, defualt=str, indent=4),
     )
+
+
+def aws_region_names():
+    """
+    Lists AWS regions
+
+    Returns:
+        list: list of AWS regions name
+
+    """
+    regions = ec2_client().describe_regions()["Regions"]
+    return [region["RegionName"] for region in regions]
+
+
+def get_least_crowded_aws_vpc_region(region_list):
```

Comment:
the function should return least crowded from all regions; call `aws_region_names` from within the function

Reply:
For this function in [managed-services-integration-tests](https://github.com/RedHatQE/managed-services-integration-tests/blob/main/tests/hypershift/test_hypershift.py#L39), there is different regions passed, rosa_regions(https://github.com/RedHatQE/managed-services-integration-tests/blob/main/tests/hypershift/test_hypershift.py#L23), that's why I added the param here instead of direcly using `aws_region_names`